pages/linux/fcrackzip.md

```diff
@@ -0,0 +1,27 @@
+# fcrackzip
+
+> ZIP archive password cracking.
+
+- Bruteforce password with length 4 to 8 with only lowercase, uppercase and digit characters (order matters):
+
+`fcrackzip -b -l 4-8 -c aA1 {{archive}}`
+
+- Bruteforce password with length 3 with only lowercase characters, $ and % (verbose mode):
```

Comment:
Perhaps this will read better?
```suggestion
- Brute-force a password in verbose mode with a length of 3 characters that only contains lowercase characters, `$` and `%`:
```